connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorPluginsResourceTest.java

```diff
@@ -182,4 +194,29 @@ public ConfigDef config() {
             return CONFIG_DEF;
         }
     }
+
+    private static class IntegerRecommender implements Recommender {
+
+        @Override
+        public List<Object> validValues(String name, Map<String, Object> parsedConfig) {
+            return Arrays.<Object>asList(1, 2, 3);
+        }
+
+        @Override
+        public boolean visible(String name, Map<String, Object> parsedConfig) {
+            return true;
+        }
+    }
+
+    private static class ListRecommender implements Recommender {
+        @Override
+        public List<Object> validValues(String name, Map<String, Object> parsedConfig) {
+            return Arrays.<Object>asList("a", "b", "c");
```

Comment:
Shouldn't this be something more like `Arrays.<Object>asList(Arrays.asList("a", "b"), Arrays.asList("c"), Arrays.asList("e","f"))` since it should be a list of the requested type, i.e. a list of lists?